apis/r/R/TileDBObject.R

```diff
@@ -7,31 +7,22 @@
 TileDBObject <- R6::R6Class(
   classname = "TileDBObject",
   public = list(
-    #' @field verbose Whether to print verbose output
-    verbose = TRUE,
-    #' @field config optional configuration
-    config = NULL,
-    #' @field ctx optional tiledb context
+    #' @field platform_config Optional platform configuration
+    platform_config = NULL,
+    #' @field ctx Optional TileDB context
     ctx = NULL,
 
     #' @description Create a new TileDB object.
     #' @param uri URI for the TileDB object
     #' @param verbose Print status messages
-    #' @param config optional configuration
+    #' @param platform_config Optional platform configuration
```

Comment:
Should we detail what we expect here?   If we can do without nesting a named character vectors works (in the context of TileDB Core config where right-hand side values are all character).

Reply:
This is a placeholder at the moment. We can round out the platform_config functionality in a future PR.